test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -133,7 +134,15 @@ u256 EwasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 		accessMemory(arg[0], 4);
 		return readMemoryHalfWord(arg[0]);
 	}
-
+	else if (_fun == "i32.clz"_yulstring)
+	{
+		// NOTE: the clz implementation assumes 64-bit inputs, hence the adjustment
+		return clz64(arg[0] & uint32_t(-1)) - 32;
+	}
+	else if (_fun == "i64.clz"_yulstring)
+	{
```

Comment:
Actually according to the coding style these also shouldn't have brackets ;-)...